src/components/Landing.js

```diff
@@ -1,27 +1,51 @@
-import React from 'react'
+import React from "react";
+import { Link } from "react-router-dom";
 
 function Landing() {
-    return (
-        <div className="landing-main-content">
-        <div className="landing-content-section">
-          <h1 className="landing-heading">
-            Build Light, Deploy Faster with
-            <span style={{color:"#FEFE00"}}> Butter UI </span>
```

Comment:
I would suggest reconsidering the yellow used for 'ButterUI', it is slightly harsh on the eyes.

Reply:
okay, Ill fix it